### ISSUE TITLE
request http://taginfo.openstreetmap.org/projects file for maps.me

### ISSUE THREAD

**Issue by ImreSamu** [mapsme/omim#2909]
As I see no taginfo project file for maps.me! 
I would like to create some (experimental) Hungarian OSM Data Quality Statistics  based on taginfo project files, 

Probably it would be 2 taginfo project files, because I would like to separete the 2 use cases:
* one for the viewer
* one for the new editor 

Maps.me is my favorite - so I would like to include to my research.  :)

It is not critical for me, and probably I can find this metadata in the source code, but a central, maintained solution would be better,    not for me but the other  taginfo users!

see more: http://wiki.openstreetmap.org/wiki/Taginfo/Projects

**Comment by deathbaba** [comment]
Our "type" tags are listed here: https://github.com/mapsme/omim/blob/master/data/mapcss-mapping.csv
Our metadata tags are here: https://github.com/mapsme/omim/blob/master/indexer/feature_meta.cpp#L45

**Comment by ImreSamu** [comment]
Thank you!   It is enought  information for me to start :)

But,  I think that   osm taginfo project file for maps.me is good idea 
so I would like to keep open this issue - with the lowest priority.

**Comment by deathbaba** [comment]
It’s hard to synchronize code with some external file.

**Comment by ImreSamu** [comment]
>It’s hard to synchronize code with some external file.

I don't understand correctly, why external file ?

for example 
* http://taginfo.openstreetmap.org/projects/opening_hours_map
 * and the JSON project file on their project github repo: https://raw.githubusercontent.com/ypid/opening_hours_map/master/taginfo.json
  * they generate with their make file : https://github.com/opening-hours/opening_hours_map/blob/master/Makefile
  * and the taginfo program download from here automaticly every day.




So as I understand, your "Maps.me taginfo" file can be any directory on your github repo. 
for example : https://raw.githubusercontent.com/mapsme/omim/master/data/taginfo.json

Just need automaticly/manually regenerate for every release.
*  or/when some meta data chenging [ data/mapcss-mapping.csv or feature_meta.cpp ]




### from the taginfo info : 

_"Integration into Taginfo
Projects have to create their project files (typically called taginfo.json) somewhere on the web. Most often it will be in their Git or other repository, but it can be anywhere on the web. The URL of the project file then has to be given to Taginfo. If you have a Github account, read https://github.com/taginfo/taginfo-projects/blob/master/CONTRIBUTING.md, add the URL to https://github.com/taginfo/taginfo-projects/blob/master/project_list.txt and send a pull request. If you don't have a github account, then email the URL to jochen@remote.org.
Taginfo will periodically poll the project file and add the information to its own database.
No project is too small or too large or too uninteresting. We want absolutely everybody's project in the database. Please consider adding your project, too!"_   http://wiki.openstreetmap.org/wiki/Taginfo/Projects

**Comment by deathbaba** [comment]
In our case, this file should be updated every time we change something in the code.

**Comment by ImreSamu** [comment]
good technical question.

i don't know if I understand you correctly,   but I will try to use 
something  that  "update  taginfo.json if taginfo.json content changed"   ...
like  `rsync  --checksum `    from http://unix.stackexchange.com/questions/29845/how-can-i-do-a-copy-if-changed-operation   ( disclaimer: not tried )

**Comment by deathbaba** [comment]
No, I mean we process some tags by our C++ code. When we add new tags there, we should manually update json file too.

**Comment by goldfndr** [comment]
A significant restriction to a taginfo project file for maps.me is that [a commit](https://github.com/mapsme/omim/commit/0c272d3be53f90907b6b20976e1b040c7bf49a75) stripped the XML information from the SVG files, preventing them from displaying graphically in GitHub and [elsehow](https://cdn.jsdelivr.net/gh/mapsme/omim@master/data/styles/clear/style-clear/symbols-ad/03005_cafe-l.svg). If there's a gateway through which the SVGs could be transformed to be graphically viewable in a web browser (or if they are graphically available individually elsewhere), that'd enable them to be included in the icon_url values of the project file (generated from the styles files).
